File: server/Audi/Entities/Homepage.cs
using System.Collections.Generic;

namespace Audi.Entities
{
    public class Homepage
    {
        public int Id { get; set; }
        public string Language { get; set; }
        public ICollection<HomepageCarouselItem> CarouselItems { get; set; }
        public int[] FeaturedProductIds { get; set; }
    }
}

File: client/projects/data/src/lib/models/homepage.ts
import { LanguageCode } from '../enums';
import { HomepageCarouselItem } from './homepage-carousel-item';
import { Product } from './product';

export interface Homepage {
  id: number;
  language: LanguageCode;
  carouselItems: HomepageCarouselItem[];
  featuredProducts: Product[];
}
